rinex/src/lib.rs

```diff
@@ -1,7 +1,5 @@
-//! This library provides a set of tools to parse, analyze
-//! and process RINEX files.
-//! Refer to README and documentation hosted here
-//! <https://github.com/georust/rinex>
+#![doc(html_logo_url = "https://raw.githubusercontent.com/georust/meta/master/logo/logo.png")]
```

Comment:
Is this what people do these days? I don't think raw.githubusercontent.com was supposed to be used as a CDN, and e.g. it doesn't set the `Content-Type` on the response.

Reply:
that's what georust::geo does at least ?

Reply:
I don't think it should, but fair enough :-).